src/js/components/Table/StyledTable.js

```diff
@@ -47,6 +47,7 @@ const StyledTableCell = styled.td`
       props.theme,
     )}
   ${props => props.tableContextTheme && props.tableContextTheme.extend}
+  ${props => props.pinnedDataTableTheme && props.pinnedDataTableTheme.extend}
```

Comment:
Doesn't seem right to have a data table theme element inside StyledTable.js

Reply:
Whoops this should be removed now. Thanks for the catch.